README.md

```diff
@@ -35,9 +35,20 @@ Usual use cases include:
         * [Tests](#RabbitMqTests)
         * [Local development](#RabbitMqLocalDevelopment)
         * [Docker image](#RabbitMqDockerImage)
+    * [ClickHouse](#ClickHouse)
```

Comment:
Link doesn't work, it jumps to news section.
Probably best to move news to changelog like it's done in querydsl here: https://github.com/infobip/infobip-spring-data-querydsl#Changelog

Reply:
yup doesn't work also for MSSQL, but usage is right below change log so issue is not that visible

Reply:
changelog moved